src/Components/Generator/RequestTypeChooser.tsx

```diff
@@ -0,0 +1,40 @@
+import t from '../../Utility/i18n';
+import Radio from '../Radio';
+import { RequestType, REQUEST_TYPES } from 'request';
+import type { JSX } from 'preact';
+
+type RequestTypeChooserProps = {
+    onTypeChange: (value: RequestType) => void;
+    current?: RequestType;
+    request_types?: RequestType[];
+};
+
+export default function RequestTypeChooser(props: RequestTypeChooserProps) {
```

Comment:
I've started to only use named exports. The benefits aren't too important but here's some reasoning:

* https://blog.neufund.org/why-we-have-banned-default-exports-and-you-should-do-the-same-d51fdc2cf2ad
* https://www.reddit.com/r/reactjs/comments/dj4mjh/react_components_named_exports_vs_default_export/

Reply:
Ah, thank you for the context. I will change that!

Reply:
Let's make this a policy. Can we enforce this via the linter?